api/external/envoy/api/v2/route/route_components.proto

```diff
@@ -0,0 +1,1624 @@
+syntax = "proto3";
```

Comment:
This proto is the replacement for `route.proto` in Gloo https://github.com/solo-io/gloo/pull/4777/files#r639104460